contributor-docs/adrs/adr-004-children-as-api.md

```diff
@@ -0,0 +1,416 @@
+# ADR 004: Strict props or Composite components
+
+## Status
+
+Approved 2022-05-10
+
+<br/>
+
+_Note: Consumer is used multiple times on this page. It refers to the developers consuming the component API and not end users._
+
+<br/>
+
+## Decision:
+
+
+1. Prefer using children for “content”
```

Comment:
Should this be its own ADR instead of being in this one?

Reply:
I think it's fine to include it in this one 🤷